components/Blog/blog.scss

```diff
@@ -0,0 +1,68 @@
+.blog-page {
+  display: flex;
+}
+
+.side-nav-blog {
+  float: left;
```

Comment:
Dont think you'll need to use `float` at all with `display: flex` in the main div

Reply:
I suggest making this a fixed width something like 30-40rem